src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -52,8 +55,8 @@ public static void GetMinMaxThreadsTest()
         {
             int minw, minc;
             ThreadPool.GetMinThreads(out minw, out minc);
-            Assert.True(minw > 0);
-            Assert.True(minc > 0);
+            Assert.True(minw >= 0);
+            Assert.True(minc >= 0);
```

Comment:
I think these should return Environment.ProcessorCount if min/max thread changes are not supported in uapaot